### ISSUE TITLE
Enable hmail_spamfilter on by default for new accounts

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#90]
```
Please share settings that will automatically enable hmail_spamfilter by 
default when a new account is registered.
```

Original issue reported on code.google.com by `jeffsh...@gmail.com` on 12 Aug 2010 at 2:25

**Comment by GoogleCodeExporter** [comment]
```
Check this version. It should enable spamfilter when a new user is added to 
roundcube database (first time login).
```

Original comment by `rol...@liebl.ath.cx` on 12 Aug 2010 at 9:02
- Changed state: **Accepted**

Attachments:
- [hmail_spamfilter.php](https://storage.googleapis.com/google-code-attachments/myroundcube/issue-90/comment-1/hmail_spamfilter.php)

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 12 Aug 2010 at 2:21

**Comment by GoogleCodeExporter** [comment]
```
No feedback. Fixed in 3878.1.
```

Original comment by `rol...@liebl.ath.cx` on 14 Aug 2010 at 4:54

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 14 Aug 2010 at 4:54
- Changed state: **Fixed**
